### PR TITLE
Homepage for the movers and the transportation companies

### PR DESCRIPTION
**Pull request by Nicanor008**
#### What does this PR do?
- Displays specific details on movers homepage

#### Description of Task to be completed?
- Displays users who have selected that movers
- Display all users who need relevant services
- Displays available drivers

#### Background Context
- Context separation between the user and the mover

#### Screenshot
Screenshot 2019-10-03 at 17 52 53
Screenshot 2019-10-03 at 17 53 00

### Message when the mover has not been selected by any user
Screenshot 2019-10-03 at 16 15 17

### CODE REVIEW

File: src/containers/Driver/HomePage/availableDrivers.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        className="IsLoggedinHomePageWrapper availableDriverMovers"
+      >
+        <Container>
+          <div className="AvailableDrivers">
+            <center>{data.length > 0 && "Available Drivers"}</center>
+          </div>
+
+          <br />
+          <br />
+
```
Comment (joyjoie) [inline src/containers/Driver/HomePage/availableDrivers.js]: remove extra spaces in the line breaks

File: src/containers/Driver/HomePage/availableUsers.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      const { data } = this.props;
+    return (
+      <div className="IsLoggedinHomePageWrapper">
+        <Container>
+          <div className="AvailableDrivers">
+            <center>
+              {data.length > 0 && "All Users in need of Moving Services"}
+            </center>
+          </div>
+
```
Comment (joyjoie) [inline src/containers/Driver/HomePage/availableUsers.js]: remove extra spaces in the line breaks and **indent  your code** to maintain uniformity

File: src/containers/Driver/HomePage/selectedMovers.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+                    {userSelectedMover.length > 0
+                      ? "Users who need your Services"
+                      : "No User has requested for your services. Be Sure to update your profile to be seen by users"}
+                  </center>
+                </div>
+
+                <br />
+                <br />
+
+                <div>
```
Comment (joyjoie) [inline src/containers/Driver/HomePage/selectedMovers.js]: remove extra line spaces and indent code

File: src/containers/Driver/HomePage/availableDrivers.js (hunk 1 above)
Comment (Nicanor008) [reply src/containers/Driver/HomePage/availableDrivers.js]: implemented

File: src/containers/Driver/HomePage/availableUsers.js (hunk 2 above)
Comment (Nicanor008) [reply src/containers/Driver/HomePage/availableUsers.js]: implemented

File: src/containers/Driver/HomePage/selectedMovers.js (hunk 3 above)
Comment (Nicanor008) [reply src/containers/Driver/HomePage/selectedMovers.js]: implemented

**Pull request merged**